cmd/generate.go

```diff
@@ -187,7 +188,11 @@ func (gc *generateCmd) run() error {
 		log.Fatalf("error in SetPropertiesDefaults template %s: %s", gc.apimodelPath, err.Error())
 		os.Exit(1)
 	}
-	template, parameters, err := templateGenerator.GenerateTemplate(gc.containerService, engine.DefaultGeneratorCode, BuildTag)
```

Comment:
ditto above

kindly leave the proposed "cutover" implementation in comments so we can easily create a test branch and continue to gather functional test data in preparation for the "real" cutover